src/Evolution/Systems/GeneralizedHarmonic/Equations.hpp

```diff
@@ -71,14 +71,7 @@ struct ComputeDuDt {
       ::Tags::deriv<Tags::Pi<Dim>, tmpl::size_t<Dim>, Frame::Inertial>,
       ::Tags::deriv<Tags::Phi<Dim>, tmpl::size_t<Dim>, Frame::Inertial>,
       Tags::ConstraintGamma0, Tags::ConstraintGamma1, Tags::ConstraintGamma2,
-      Tags::GaugeH<Dim>, Tags::SpacetimeDerivGaugeH<Dim>, gr::Tags::Lapse<>,
-      gr::Tags::Shift<Dim>, gr::Tags::InverseSpatialMetric<Dim>,
-      gr::Tags::InverseSpacetimeMetric<Dim>,
-      gr::Tags::TraceSpacetimeChristoffelFirstKind<Dim>,
-      gr::Tags::SpacetimeChristoffelFirstKind<Dim>,
-      gr::Tags::SpacetimeChristoffelSecondKind<Dim>,
-      gr::Tags::SpacetimeNormalVector<Dim>,
-      gr::Tags::SpacetimeNormalOneForm<Dim>>;
+      Tags::GaugeH<Dim>, Tags::SpacetimeDerivGaugeH<Dim>>;
```

Comment:
Would be nice, as long as we're cleaning this code up, to fix the help text to either cite the exact equations or, better yet, include the RHS equations using our standard notation right in the help text.

Reply:
I can do this, I have them in TeX anyway. I'll also add some other constraint damping parameters from other papers and note that they are currently zero but that we should considering exploring them in the future :)